controllers/tenants.rb

```diff
@@ -0,0 +1,17 @@
+get '/apartments/:id/tenants' do
```

Comment:
how are you observing your apartment views you created? seems like there's no controller for apartments yet.